tests/conformance/scripts/run-tests/runners/index.ts

```diff
@@ -6,9 +6,9 @@ import TypescriptStreamedTestRunner from "./TypescriptStreamedTestRunner";
 export type { ITestRunner };
 
 const runners: readonly ITestRunner[] = [
-  new TypescriptStreamedTestRunner(),
-  new TypescriptIndexedTestRunner(),
   new PythonStreamedTestRunner(),
+  new TypescriptIndexedTestRunner(),
+  new TypescriptStreamedTestRunner(),
```

Comment:
we have two streamed runners now, can remove one

Reply:
Oops thanks for catching that.